app/views/posts/show.html.erb

```diff
@@ -1,14 +1,40 @@
-<p>
-  <strong>Title:</strong>
-  <%= @post.title %>
-</p>
- 
-<p>
-  <strong>Text:</strong>
-  <%= @post.text %>
-</p>
+<div class="container">
+  <p>
+    <strong>Title:</strong>
+    <%= @post.title %>
+  </p>
+  
+  <p>
+    <strong>Text:</strong>
+    <%= @post.text %>
+  </p>
 
 <p>
   <strong>Author:</strong>
   <%= @post.user.name %>
-</p>
\ No newline at end of file
+</p>
+
+<h2>Comments</h2>
+<% @post.comments.each do |comment| %>
+  <p>
+    <strong>Commenter:</strong>
+    <%= current_user.name %>
```

Comment:
will the commenter always be the current user?

Reply:
Yes, when you create a comment you're logged in your account, so the author of the comment is the current user. Right?

Reply:
yes but you list all comments here, all users should be able to comment right

Reply:
of course, you're totally right!! (again)! It's fixed now!